crates/turborepo-ui/src/tui/popup.rs

```diff
@@ -0,0 +1,60 @@
+use ratatui::{
+    layout::{Constraint, Flex, Layout, Rect},
+    text::Line,
+    widgets::{Block, List, ListItem, Padding},
+};
+
+use super::size::SizeInfo;
+
+const BIND_LIST_ITEMS: [&str; 11] = [
+    "m - Toggle this help popup",
+    "↑ or j - Select previous task",
+    "↓ or k - Select next task",
+    "h - Toggle task list visibility",
+    "/ - Filter tasks to search term",
+    "ESC - Clear filter",
+    "i - Interact with task",
+    "CTRL+z - Stop interacting with task",
```

Comment:
is it `CTRL` on windows and linux and mac?

Reply:
I can at least confirm its `CTRL` on Mac. Lines like [this one](https://github.com/vercel/turborepo/blob/e901df637d9fdee3a91907a2213d7f8cc01a45ef/crates/turborepo-ui/src/tui/input.rs#L63) and [this one](https://github.com/vercel/turborepo/blob/e901df637d9fdee3a91907a2213d7f8cc01a45ef/crates/turborepo-ui/src/tui/input.rs#L71) lead me to believe they should be cross-platform but @chris-olszewski to confirm as I'm figuring he considered this when setting things up.